src/client/accounts.rs

```diff
@@ -76,26 +76,24 @@ impl<N: NodeRpcClient, D: DataStore> Client<N, D> {
             AuthData::RpoFalcon512Seed(key_pair) => {
                 let keypair = KeyPair::from_seed(&key_pair)?;
                 match (account_data.account.is_new(), account_data.account_seed) {
-                    (true, Some(seed)) => self.insert_account(
-                        &account_data.account,
-                        seed,
-                        &AuthInfo::RpoFalcon512(keypair),
-                    ),
+                    (true, None) => Err(ClientError::ImportNewAccountWithoutSeed),
                     (false, Some(seed)) => {
                         tracing::warn!(
                             "Imported an existing account and still provided a seed when it is not needed. It's possible that the account's file was incorrectly generated."
                         );
 
                         self.insert_account(
                             &account_data.account,
-                            seed,
+                            Some(seed),
                             &AuthInfo::RpoFalcon512(keypair),
                         )
                     }
-                    (false, None) => {
-                        unimplemented!();
-                    }
-                    (true, None) => Err(ClientError::ImportNewAccountWithoutSeed),
+                    // Cases left: non-new account with no seed, and new account with seed; both OK
```

Comment:
```suggestion
                    // Non-new account with no seed, and new account with seed; both OK
```

Can we keep it like this? I think the "Cases left" might get confused with cases we're missing